server/src/main/java/com/ahmadda/domain/EventOperationPeriod.java

```diff
@@ -63,6 +63,21 @@ public boolean canNotRegistration(final LocalDateTime currentDateTime) {
         return !registrationPeriod.includes(currentDateTime);
     }
 
+    public void closeRegistration(final LocalDateTime closeTime) {
+        Period closePeriod = Period.create(this.registrationPeriod.start(), closeTime);
+
+        validateClosePeriod(closeTime);
+        validate(closePeriod, this.eventPeriod);
+
+        this.registrationPeriod = closePeriod;
+    }
+
+    private void validateClosePeriod(final LocalDateTime closeDateTime) {
+        if (closeDateTime.isAfter(this.registrationPeriod.end())) {
+            throw new BusinessRuleViolatedException("마감 시간은 현재 등록 종료 시간보다 이전이어야 합니다.");
+        }
+    }
```

Comment:
p2
예외 메시지를 다음과 같이 하는게 더 명확해 보이는데 어떻게 생각하시나요!
그리고 저희 도메인 용어가 등록이 아닌 신청 입니다!
```suggestion
    private void validateClosePeriod(final LocalDateTime closeDateTime) {
        if (closeDateTime.isAfter(this.registrationPeriod.end())) {
            throw new BusinessRuleViolatedException("이미 신청이 마감된 이벤트입니다.");
        }
    }
```

Reply:
굿 인정합니다.